src/XIVLauncher/Windows/CustomMessageBox.xaml.cs

```diff
@@ -31,7 +32,7 @@ private CustomMessageBox()
             Focus();
         }
 
-        public static void Show(string text, string caption, MessageBoxButton buttons = MessageBoxButton.OK, MessageBoxImage image = MessageBoxImage.Asterisk, bool showHelpLinks = true)
+        public static void Show(string text, string caption, MessageBoxButton buttons = MessageBoxButton.OK, MessageBoxImage image = MessageBoxImage.Asterisk, bool showHelpLinks = true, bool showReportLinks = false)
```

Comment:
This is not great, I'd rather add a way to just define buttons and actions, but it'll have to do for now